src/StripeTests/Services/Accounts/AccountServiceTest.cs

```diff
@@ -34,6 +34,17 @@ public class AccountServiceTest : BaseStripeTest
                     Name = "business name",
                 },
                 BusinessType = "company",
+                Capabilities = new AccountCapabilitiesOptions
```

Comment:
Is this change necessary with the bump of stripe-mock version? (Just wondering why it's present in this PR)

Reply:
Yes! `RequestedCapabilities` is deprecated and was replaced with `Capabilities` last week so when we bumped stripe-mock` we had to make that change and change the `Type` on `AccountLink` too